docs/content/deployment/dagster-instance.mdx

```diff
@@ -459,36 +666,36 @@ local_artifact_storage:
 
 ### Telemetry
 
-This allows opting in/out (set to `true` by default) of Dagster collecting anonymized usage statistics.
-
-To configure Telemetry, set the `telemetry` key in your `dagster.yaml`.
-
-For more information on how and why we use telemetry, please visit the <Link href="/getting-started/telemetry">Telemetry Docs</Link>.
+The `telemetry` key allows you to opt in or out of Dagster collecting anonymized usage statistics. This is set to `true` by default.
 
 ```yaml file=/deploying/dagster_instance/dagster.yaml startafter=start_marker_telemetry endbefore=end_marker_telemetry
 # Allows opting out of Dagster collecting usage statistics.
 telemetry:
   enabled: false
 ```
 
+Refer to the [Telemetry documentation](/getting-started/telemetry) for more info.
+
 ### Code servers
 
-The `code_servers` key lets you configure how Dagster loads the code in your [workspace](/concepts/code-locations/workspace-files).
+The `code_servers` key allows you to configure how Dagster loads the code in a [code location](/concepts/code-locations/workspace-files).
```

Comment:
Unfortunate that code server is the config key as it adds another slightly different verbiage.

Reply:
I think it might be more clear to call this section "grpc servers" since this is behavior it really controls. However I do not feel super strongly.

Reply:
That's fine with me - not all of the other headings match up with key names.